cliquet/collection.py

```diff
@@ -0,0 +1,235 @@
+class Collection(object):
+    """A collection stores and manipulate records in its attached storage.
+
+    It is not aware of HTTP environment nor protocol.
+
+    Records are isolated according to the provided `name` and `parent_id`.
+
+    Those notions have no particular semantic and can represent anything.
+    For example, the collection `name` can be the *type* of objects stored, and
+    `parent_id` can be the current *user id* or *a group* where the collection
+    belongs. If left empty, the collection records are not isolated.
+    """
+    id_field = 'id'
+    """Name of `id` field in records"""
+
+    modified_field = 'last_modified'
+    """Name of `last modified` field in records"""
+
+    deleted_field = 'deleted'
+    """Name of `deleted` field in deleted records"""
+
+    def __init__(self, storage, id_generator=None, collection_id='',
+                 parent_id='', auth=None):
+        """
+        :param storage: an instance of storage
+        :type storage: :class:`cliquet.storage.Storage`
+        :param id_generator: an instance of id generator, used by storage
```

Comment:
okay, got it. All collections aren't using the same so it makes sense to have it here.